src/renderers/dom/client/wrappers/__tests__/ReactDOMSelect-test.js

```diff
@@ -454,4 +454,15 @@ describe('ReactDOMSelect', function() {
     expect(node.options[1].selected).toBe(false);  // giraffe
     expect(node.options[2].selected).toBe(false);  // gorilla
   });
+
+  it('should throw warning message if value is null', function() {
+    spyOn(console, 'error');
+
+    var link = new ReactLink(null, mocks.getMockFunction());
+    ReactTestUtils.renderIntoDocument(<select valueLink={link}><option value="test"/></select>);
+    expect(console.error.argsForCall.length).toBe(1);
+
```

Comment:
We should assert that the console.error contains the expected error message.  See how I did this in https://github.com/facebook/react/pull/5032/files